package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
hey @ohadlevy @jtomasek, i've heard others say they have a preference for using `optionalDependencies` for third party libraries that are not a requirement for all library components. From the NPM documentation:
https://docs.npmjs.com/files/package.json#optionaldependencies
> If a dependency can be used, but you would like npm to proceed if it cannot be found or fails to install, then you may put it in the optionalDependencies object. This is a map of package name to version or url, just like the dependencies object. The difference is that build failures do not cause installation to fail.

I suppose it just depends on how we look at React Bootstrap. It will likely be used in several components so I don't have an issue making it a hard dependency (and thus making the build fail without it).

Regarding "it does not land in your bundle if not imported", yes, that's correct if you are using Webpack2.
https://webpack.js.org/guides/tree-shaking/

So on the client, it will have no impact if not imported. It will only have an impact on the size of your OCP build image / container image (if a hard dependency in this project on your OCP build).

Hope my thought process is making sense ;)